tests/test_h1_client.c

```diff
@@ -216,7 +216,7 @@ static enum aws_http_outgoing_body_state s_simple_send_body(
     void *user_data) {
 
     (void)stream;
-    struct simple_body_sender *data = user_data;
+    struct simple_body_sender *data = (struct simple_body_sender *)user_data;
```

Comment:
This change shouldn't be necessary

Reply:
Oh, that is just because the VScode give me an error on that...